codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/GoDependency.java

```diff
@@ -38,21 +38,27 @@
     NET_HTTP("stdlib", "", "net/http", null, Versions.GO_STDLIB),
     BYTES("stdlib", "", "bytes", null, Versions.GO_STDLIB),
     STRINGS("stdlib", "", "strings", null, Versions.GO_STDLIB),
+    JSON("stdlib", "", "encoding/json", null, Versions.GO_STDLIB),
     IO("stdlib", "", "io", null, Versions.GO_STDLIB),
+    IOUTIL("stdlib", "",  "io/ioutil", null, Versions.GO_STDLIB),
 
     SMITHY("dependency", "github.com/awslabs/smithy-go",
             "github.com/awslabs/smithy-go", "smithy", Versions.SMITHY_GO),
     SMITHY_HTTP_TRANSPORT("dependency", "github.com/awslabs/smithy-go",
             "github.com/awslabs/smithy-go/transport/http", "smithyhttp", Versions.SMITHY_GO),
     SMITHY_MIDDLEWARE("dependency", "github.com/awslabs/smithy-go",
             "github.com/awslabs/smithy-go/middleware", null, Versions.SMITHY_GO),
+    SMITHY_TIME("dependency", "github.com/awslabs/smithy-go",
+            "github.com/awslabs/smithy-go/time", "smithytime", Versions.SMITHY_GO),
 
     AWS_REST_PROTOCOL("dependency", "github.com/aws/aws-sdk-go-v2",
             "github.com/aws/aws-sdk-go-v2/aws/protocol/rest", null, Versions.AWS_SDK),
     AWS_PRIVATE_PROTOCOL("dependency", "github.com/aws/aws-sdk-go-v2",
             "github.com/aws/aws-sdk-go-v2/private/protocol", null, Versions.AWS_SDK),
     AWS_JSON_PROTOCOL("dependency", "github.com/aws/aws-sdk-go-v2",
-            "github.com/aws/aws-sdk-go-v2/aws/protocol/json", null, Versions.AWS_SDK);
+            "github.com/aws/aws-sdk-go-v2/aws/protocol/json", null, Versions.AWS_SDK),
+    AWS_JSON_PROTOCOL_ALIAS("dependency", "github.com/aws/aws-sdk-go-v2",
+            "github.com/aws/aws-sdk-go-v2/aws/protocol/json", "jsonprotocol", Versions.AWS_SDK);
 
```

Comment:
Does it make sense to always use the same pkg alias for this so don't have to be concerned with which one will be used? e.g. maybe just `awsjson`  Which i think will be replaced with `smithyjson` in another PR

Reply:
Can do that. Will prefer to make this change when we move the package to smithy, to avoid any confusions.